Code/Framework/AzToolsFramework/AzToolsFramework/API/ToolsApplicationAPI.h

```diff
@@ -605,6 +605,18 @@ namespace AzToolsFramework
          * system simulates its execution, and avoids some code duplication.
          */
         virtual void RunRedoSeparately(UndoSystem::URSequencePoint* redoCommand) = 0;
+
+        /**
+         * Create a prefab out of the entities provided, at the path provided.
+         * Automatically detects descendants of entities, and discerns between entities and child instances.
+         */
+        virtual bool CreatePrefab(
+            const AZStd::vector<AZ::EntityId>& entityIds, const char* filePath, bool saveToDisk) = 0;
```

Comment:
Prefer AZ::IO::Path, however, is that exposed to python? If not AZStd::string_view can also work here

Reply:
I didn't see that AZ::IO::PathView is exposed to python. I will use string_view here.